GDJS/Runtime/gd.js

```diff
@@ -33,35 +33,50 @@ window.gdjs = {
  * Convert a rgb color value to a hex string.
  *
  * No "#" or "0x" are added.
+ * @param {number} r Red
+ * @param {number} g Green
+ * @param {number} b Blue
+ * @returns {string}
  */
 gdjs.rgbToHex = function(r, g, b) {
   return '' + ((1 << 24) + (r << 16) + (g << 8) + b).toString(16).slice(1);
 };
 
 /**
  * Convert a rgb color value to a hex value.
+ * @param {number} r Red
+ * @param {number} g Green
+ * @param {number} b Blue
+ * @returns {number}
  */
 gdjs.rgbToHexNumber = function(r, g, b) {
   return (r << 16) + (g << 8) + b;
 };
 
 /**
  * Get a random integer between 0 and max.
+ * @param {number} max The maximum value.
+ * @returns {number}
  */
 gdjs.random = function(max) {
   if (max <= 0) return 0;
   return Math.floor(Math.random() * (max + 1));
 };
 
 /**
- * Get a random integer between min and max
+ * Get a random integer between min and max.
+ * @param {number} min The minimum value.
+ * @param {number} max The maximum value.
+ * @returns {number}
  */
 gdjs.randomInRange = function(min, max) {
   return min + gdjs.random(max - min); // return min if min >= max
 };
 
 /**
  * Get a random float between 0 and max.
```

Comment:
Let's make this description a bit more complete:

Get a random float in the range 0 to less than max (inclusive of 0, but not max).

Reply:
Ping :)